docs/2-deployments/7-multitenant-deployments.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Multi-Tenant Deployments
+---
+
+# Multi-Tenant Deployments
+
+Multi-tenant deployments are when
+
+By default, deploying a component will create it as a singleton in the environment, where it is the default instance.
+
+In some cases you may want to deploy multiple instances of a component to an environment. The deployment process creates a global tenant by default, but you can create a named tenant simply by naming the component being deployed. This is called a "multi-tenant deployment".
+
+```
+architect-local deploy examples/react-app@app1
+```
+
+This process works for updating existing deployments as well. If an "app1" tenant of the `examples/react-app` component had already been deployed, the above command would have deployed to that same tenant, leaving any other instances of the component untouched.
+
+## Deployments and Management Lifecycle
+
+When you view your deployed components, you'll see that the react-app has the name "app1" whereas the typical deployment without a tenant name will show only the component. In this example we've deployed the example React App twice with the tenant names "app1" and "app2", and once without a tenant name.
```

Comment:
"When you view your multi-tenant components..."

Reply:
I just meant to refer to the list of components that are deployed to the environment, not just the multi-tenant deployments. I could rephrase the start of this to say The "Environment's components list page" instead if you think that communicates the point better?